annotations/src/main/java/org/corfudb/runtime/object/IObjectBuilder.java

```diff
@@ -8,7 +8,7 @@
  * @param <T> The type of the object to build.
  * Created by mwei on 11/12/16.
  */
-public interface IObjectBuilder<T> {
+public interface IObjectBuilder<T extends ISMRObject<T>> {
```

Comment:
On master this should have been `T extends ICorfuSMR`, if you look at `getWrapper` the type will actually get recasted as `(Class<ICorfuSMR<T>>)`. Look at the next comment: this change shouldn't be necessary.

Reply:
I see.

Reply:
Fixed. `ISMRObject` does not exist anymore.